frontend/src/pages/PipelineDetails.test.tsx

```diff
@@ -507,20 +514,6 @@ describe('PipelineDetails', () => {
     },
   );
 
-  it('uses an empty string and does not show error when getTemplate response is empty', async () => {
```

Comment:
Can we change the implementation of this test instead of removing it? We want to validate the behavior when `Apis.pipelineServiceApiV2.getPipelineVersion` doesn't return a valid pipeline_spec. (for example: pipeline_spec is empty)